src/MiniCover/HitServices/HitService.cs

```diff
@@ -0,0 +1,90 @@
+﻿using MiniCover.HitServices;
+using System;
+using System.Collections.Concurrent;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Threading;
+
+namespace MiniCover
+{
+    public static class HitService
```

Comment:
I think a single HitService with toggles to enable features would be better.
Soon new features will appear and we will start to duplicate code on both HitServices.